collector/monitors/hub_state_monitor.go

```diff
@@ -18,17 +18,29 @@ var (
 	BlunaExchangeRate MetricName = "bluna_exchange_rate"
 )
 
-func NewHubStateMonitor(cfg config.CollectorConfig, logger *logrus.Logger) HubStateMonitor {
-	m := HubStateMonitor{
-		State:      &types.HubStateResponseV1{},
+func NewHubStateMonitor(cfg config.CollectorConfig, logger *logrus.Logger) Monitor {
+
+	if cfg.BassetContractsVersion == config.V1Contracts {
```

Comment:
Please change to a switch with default error ("version unknown"), it's more explicit